### PR TITLE
[vrops-exporter] Splitted datastore alert in critical and warning

### PR DESCRIPTION
**Pull request by christopherhans**
https://issues.wdf.sap.corp/browse/CCM-14564

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/datastore.alerts [hunk 1]
```diff
@@ -3,13 +3,28 @@ groups:
   rules:
   - alert: DataStoreCapacityWarning
     expr: >
-      max_over_time(vrops_datastore_diskspace_total_usage_gigabytes[10m]) / max_over_time(vrops_datastore_diskspace_capacity_gigabytes[10m]) > 0.9
+      max_over_time(vrops_datastore_diskspace_total_usage_gigabytes[10m]) / max_over_time(vrops_datastore_diskspace_capacity_gigabytes[10m]) >= 0.8 < 9
```
Comment (Carthaca) [inline prometheus-exporters/vrops-exporter/alerts/datastore.alerts:6]: FYI `>= 0.8` would work as well (without the `< 9`, which should be **0.9** if you don't omit it)

There is the concept of alert inhibitions: if alerts of the same context are firing (in this case warning >=0.8 and critical >0.9) only the one with the higher severity will create a notification.

File: prometheus-exporters/vrops-exporter/alerts/datastore.alerts [hunk 2]
```diff
@@ -3,13 +3,28 @@ groups:
...
       context: "vrops-exporter"
+      meta: "Datastore {{ $labels.datastore }} utilization > 80% ({{ $labels.datacenter }}, {{ $labels.vccluster }}, {{ $labels.hostsystem }})"
+      dashboard: vcenter-datastore-utilization
+      playbook: docs/support/playbook/datastore/datastorediskusagealarm.html
+    annotations:
+      description: "Datastore {{ $labels.datastore }} utilization > 80% ({{ $labels.datacenter }}, {{ $labels.vccluster }}, {{ $labels.hostsystem }})"
+      summary: "Datastore {{ $labels.datastore }} utilization > 80% ({{ $labels.datacenter }}, {{ $labels.vccluster }}, {{ $labels.hostsystem }})"
+  - alert: DataStoreCapacityCritical
+    expr: >
+      max_over_time(vrops_datastore_diskspace_total_usage_gigabytes[10m]) / max_over_time(vrops_datastore_diskspace_capacity_gigabytes[10m]) >=  9
```
Comment (Carthaca) [inline prometheus-exporters/vrops-exporter/alerts/datastore.alerts:21]: 0.9

**Review by Carthaca — CHANGES_REQUESTED** [review]
Hi @christopherhans, thanks for working on this - please see my inline comments

File: prometheus-exporters/vrops-exporter/alerts/datastore.alerts (hunk 1 above)
Comment (christopherhans) [reply prometheus-exporters/vrops-exporter/alerts/datastore.alerts]: Thanks for the hint

**Review by Carthaca — APPROVED** [review]
👍

**Pull request merged**